STYLEGUIDE.md

```diff
@@ -71,10 +72,11 @@ Must (not), should (not), recommended, suggested, may, and can generally follow
 ## Images/Figures
 
 - Lower case file names and extensions, separate with hyphens, 3-character extensions
-- For ease of visibility, explanatory images should be full width; expository or those with very simple subjects can be half-width, right aligned
+- For ease of visibility, images should be either full width, or 500 px if appropriate
 - Use PNG for all images, except photographs over 100-200 KB as a PNG (use moderately compressed JPEG); avoid sizes above 2000 x 2000 or below 1000 x 1000 if full width, or half of each if half width
+- GIFs should be compressed to 5 frames/s and should target a length of around 5-10 seconds, and be 500 px unlesss needed to illustrate a more complex or important subject
```

Comment:
A lot of the gifs we already have are greater than 500 px. I would say most of them are 690 so this doesn't really make much sense.

Reply:
Okay, sure. Thanks for pointing it out, I'll reword this.

Reply:
Reworded